lib/tasks/dev.rake

```diff
@@ -0,0 +1,51 @@
+# Don't add any includes that aren't in the Ruby std-lib here,
```

Comment:
We already have the vagrant box and this is duplicating bootstrap.sh a lot. Too much for my taste, sorry.